lib/routemaster.dart

```diff
@@ -224,9 +227,9 @@ class Routemaster {
   ///
   @optionalTypeArgs
   NavigationResult<T> push<T extends Object?>(
-    String path, {
-    Map<String, String>? queryParameters,
-  }) {
+      String path, {
+        Map<String, String>? queryParameters,
+      }) {
```

Comment:
hey @xoltawn , I think you don't use the same dart formatter as @tomgilder , which results in a lot of unnecessary changes :)

Reply:
I edited the files and now we have the lease changes possible.